src/components/StyleGuide/Typography.mdx

```diff
@@ -16,27 +18,85 @@ import Text from "../Text/Text";
 - {<Link href="#heading-component" target="_self">Heading Component</Link>}
 - {<Link href="#text-component" target="_self">Text Component</Link>}
 - {<Link href="#font-family" target="_self">Font Family</Link>}
-- {<Link href="#font-color" target="_self">Font Color</Link>}
+- {<Link href="#font-color-and-accessibility" target="_self">Font Color and Accessibility</Link>}
 - {<Link href="#font-weight" target="_self">Font Weight</Link>}
-- {<Link href="#font-sizes" target="_self">Font Sizes</Link>}
 - {<Link href="#text-case" target="_self">Text Case</Link>}
+- {<Link href="#line-length-and-line-height" target="_self">Line Length and Line Height</Link>}
 - {<Link href="#character-count" target="_self">Character Count</Link>}
 - {<Link href="#figma-reference" target="_self">Figma Reference</Link>}
 
 ## General Information
 
 Consuming applications should utilize the Reservoir Design System (DS) standard
-text components as much as possible. In cases where that is not possible, the
-DS exposes CSS variables to assist with formatting text elements.
+text components as much as possible. In cases where that is not possible, the DS
+exposes design tokens to assist with formatting text elements.
 
 The sizing of all text elements in the Design System is based on `1rem` being
 equal to `16px`.
 
 ## Heading Component
 
 Use the DS [Heading](../?path=/docs/components-typography-styles-heading--docs)
-component to render a standard HTML `<h>` tag. The `level` prop can be used to
-set the specific `<h>` tag that you require.
+component to render a native HTML `<h>` tag. The `level` prop can be used to set
+the specific `<h>` tag that you require.
+
+### Updated Typographic Styles
+
+New typographic styles were introduced in June 2023 and the new styles are the
+recommended options for the native `h` elements and the `size` overrides.
+
+### Default Styles for Native Heading Elements
```

Comment:
This is better repeated than linked you think?

Reply:
This is something I debated as I would putting this together.  I opted to rewrite the same content in both places.